cmd/install/install_render.go

```diff
@@ -52,7 +78,13 @@ func NewRenderCommand(opts *Options) *cobra.Command {
 
 		var objects []crclient.Object
 
-		if opts.Template {
+		if opts.SSSTemplate {
+			templateObject, err := hyperShiftOperatorSSSTemplateManifest(opts)
+			if err != nil {
+				return err
+			}
+			objects = []crclient.Object{templateObject}
+		} else if opts.Template {
```

Comment:
I don't think there's any consumer for the old template anymore? If so we can follow up to remove all of it @geoberle

Reply:
I can confirm we (app-sre) don't use it anymore (I work with @geoberle )